docs/getting_started/installation.rst

```diff
@@ -135,6 +135,22 @@ install the assets:
     bin/console cache:clear
     bin/console assets:install
 
+Webpack Encore Integration
+--------------------------
+
+When the assets are installed add following to  your configuration:
+
+.. code-block:: yaml
+
+    # config/sonata_admin.yaml
```

Comment:
```suggestion
    # config/webpack_encore.yaml
```
?

Reply:
Why not putting this in sonata_admin for the sonata specific assets? see https://github.com/symfony/recipes-contrib/pull/714